### DISCUSSION TITLE
What should the developer container contain? Whether we can prepare developer container details

### DISCUSSION THREAD

**Discussion by ywh555hhh** [apache/incubator-graphar#501] [Q&A] [answered]
The development container does not seem to contain the following content, which comes from the [cpp library readme](https://github.com/apache/incubator-graphar/tree/main/cpp)
```
Dependencies for optional features:

[Doxygen](https://www.doxygen.nl/index.html) (>= 1.8) for generating documentation
Extra dependencies are required by examples:

[BGL](https://www.boost.org/doc/libs/1_80_0/libs/graph/doc/index.html) (>= 1.58)
```

and could we provide a list of what dependencies our container contains and their version numbers

**Comment by acezen** [comment, marked as answer]
Thanks for the suggestion, the dev container image currently only contains essential dependencies for library like

- cmake/g++, arrow for c++
- jdk for java

the testing and example related dependencies has not been included yet but they should be. I would publish a new container image and add the list or DockerFile to show the dependencies the container contains as you said.
